MEME/MEME/Artist/ArtistHomeViewController.swift

```diff
@@ -8,10 +8,101 @@
+    // 내일 예약 수
+    private var tomorrowRes : Int = 3
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        UISet()
+        tableViewConfigure()
+    }
+    private func UISet(){
+        
+        if isTodayReservation{
+            artistHomeProfileNoStatusView.isHidden = true
+            artistHomeProfileStatusView.isHidden = false
+        }else{
+            artistHomeProfileNoStatusView.isHidden = false
+            artistHomeProfileStatusView.isHidden = true
+        }
+        
+        artistHomeProfileStatusView.layer.cornerRadius = 10
+        artistHomeProfileNoStatusView.layer.cornerRadius = 10
+        artistProfileImageView.layer.cornerRadius = artistProfileImageView.frame.height/2
+        artistProfileImageView.clipsToBounds = true
+        
+        
+        
+        artistHomeProfileLabel.text = "안녕하세요, 00님!\n내일 예약 " + String(tomorrowRes) + "건이 있어요."
+    }
+    
+    private func tableViewConfigure(){
+        artistReservationStatusTableView.delegate = self
+        artistReservationStatusTableView.dataSource = self
+        artistReservationStatusTableView.register(ArtistReservationStatusTableViewCell.nib(), forCellReuseIdentifier: ArtistReservationStatusTableViewCell.identifier)
+    }
 
+    @IBAction func profileImageTapped(_ sender: UIButton) {
+        let vc = ArtistProfilePreviewViewController()
+        self.navigationController?.pushViewController(vc, animated: true)
+    }
+    
+    @IBAction func portfolioManageBtnTapped(_ sender: UIButton) {
+        // 포트폴리오 관리 화면 전환
+    }
+    @IBAction func entireReservationBtnTapped(_ sender: UIButton) {
+        let vc = EntireArtistReservationManageViewController()
+        self.navigationController?.pushViewController(vc, animated: true)
     }
+    @objc func reservationManagedBtnTapped(){
+        let vc = ArtistReservationSingleManageViewController()
+        self.navigationController?.pushViewController(vc, animated: true)
+    }
+    
+}
 
+extension ArtistHomeViewController : UITableViewDelegate,UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if(makeUpNameArray.count>2)
+        {
+            return 2
+        }else{
+            return makeUpNameArray.count
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = artistReservationStatusTableView.dequeueReusableCell(withIdentifier: ArtistReservationStatusTableViewCell.identifier, for: indexPath) as? ArtistReservationStatusTableViewCell
+        cell?.makeUpNameLabel.text = makeUpNameArray[indexPath.row]
+        cell?.modelNameLabel.text = modelNameArray[indexPath.row]
+        cell?.reservationDateLabel.text = reservationDateArray[indexPath.row]
+        cell?.reservationManageBtn.addTarget(self, action: #selector(reservationManagedBtnTapped), for: .touchUpInside)
+        return cell ?? UITableViewCell()
```

Comment:
```suggestion
        guard let cell = artistReservationStatusTableView.dequeueReusableCell(withIdentifier: ArtistReservationStatusTableViewCell.identifier, for: indexPath) as? ArtistReservationStatusTableViewCell else { return UITableViewCell() }
        cell.makeUpNameLabel.text = makeUpNameArray[indexPath.row]
        cell.modelNameLabel.text = modelNameArray[indexPath.row]
        cell.reservationDateLabel.text = reservationDateArray[indexPath.row]
        cell.reservationManageBtn.addTarget(self, action: #selector(reservationManagedBtnTapped), for: .touchUpInside)
        return cell
```

와 같이 guard let 바인딩을 활용해도 좋을듯 합니당